### PR TITLE
Filiz iscreated

### PR DESCRIPTION
**Pull request by Filizbagdat**
dd

### CODE REVIEW

File: src/Group1/Test1.java [hunk 1]
```diff
@@ -0,0 +1,8 @@
+package Group1;
+
+public class Test1 {
+    public static void main(String[] args) {
+        System.out.println("test1 ");
```
Comment (Ozleblebici) [inline src/Group1/Test1.java:5]: This should be print method!

**Review by Ozleblebici — CHANGES_REQUESTED** [review]

File: src/Group1/Test1.java (hunk 1 above)
Comment (Filizbagdat) [reply src/Group1/Test1.java:5]: i have updated

Comment (Ozleblebici) [reply src/Group1/Test1.java:5]: Checked, looks good thanks

**Review by Ozleblebici — APPROVED** [review]
All good to go

**Pull request merged**